server/src/test/resources/org/kaazing/gateway/server/config/parse/data/gateway-config-symlink.xml

```diff
@@ -0,0 +1,20 @@
+<gateway-config xmlns="http://xmlns.kaazing.org/2014/09/gateway">
+
+
+    <service>
+        <name>symlink</name>
+        <accept>http://localhost:8000/</accept>
+        <accept>https://localhost:9000/</accept>
+
+        <type>directory</type>
+        
+        <properties>
+            <directory>/</directory>
+            <welcome-file>index.html</welcome-file>
+            <error-pages-directory>/error-pages</error-pages-directory>
+            <options>indexes</options>
+            <symlinks>follow</symlinks>
```

Comment:
Please update to <followSymlinks>enabled | disabled</followSymlinks> as outlined [here](https://github.com/kaazing/roadmap/issues/445)